datasets/oclar/README.md

```diff
@@ -0,0 +1,177 @@
+
+## Dataset Structure
+
+### Data Instances
+
+A typical data point comprises a `pagename` which is the name of service / location being reviewed, a `review` which is
+the review left by the user / client , and a `rating` which is a score between 1 and 5.
+
+The authors consider a review to be positive if the score is greater or equal than `3`, else it is considered negative.
+
+An example from the OCLAR data set looks as follows:
+
+```
+  "pagename": 'Ramlet Al Baida Beirut Lebanon',
+  "review": 'مكان يطير العقل ويساعد على الاسترخاء',
+  "rating": 5,
+```
+
+### Data Fields
+
+- `pagename`: string name of the service / location being reviewed
+- `review`: string review left by the user / costumer
+- `rating`: number of stars left by the reviewer. It ranges from 1 to 5.
+
+### Data Splits
+
+The data set comes in a single csv file of a total `3916` reviews :
+- `3465` are considered positive (a rating of 3 to 5)
+- `451` are considered negative (a rating of 1 or 2)
+
+## Dataset Creation
+
+### Curation Rationale
+
+This dataset was created for Arabic sentiment classification on services’ reviews in Lebanon country. 
+Reviews are about public services, including hotels, restaurants, shops, and others.
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+The data was collected from Google Reviews and [Zomato website](https://www.zomato.com/lebanon)
+
+#### Who are the source language producers?
+
+The source language producers are people who posted their reviews on Google Reviews or [Zomato website](https://www.zomato.com/lebanon).
+They're mainly Arabic speaking Lebanese people.
+
+### Annotations
+
+#### Annotation process
+
+The dataset does not contain any additional annotations
+
+#### Who are the annotators?
+
+[N/A]
+
+### Personal and Sensitive Information
+
+[N/A]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+The author's research has tackled a highly important task of sentiment analysis for Arabic language in the Lebanese
+context on 3916 reviews’ services from Google and Zomato. Experiments show three main findings: 
+1) The classifier is confident when used to predict positive reviews,
+2) while it is biased on predicting reviews with negative sentiment, and finally 
+3) the low percentage of negative reviews in the corpus contributes to the diffidence of LR.
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[N/A]
+
+## Additional Information
+
+### Dataset Curators
+
+This dataset was curated by Marwan Al Omari, Moustafa Al-Hajj from Centre for Language Sciences and Communication,
+Lebanese University, Beirut, Lebanon;  Nacereddine Hammami from college of Computer and Information Sciences, 
+Jouf University, Aljouf, KSA; and Amani Sabra from Centre for Language Sciences and Communication, Lebanese University,
+Beirut, Lebanon.
+
+### Licensing Information
+
+[N/A]
```

Comment:
can you change from `[N/A]` to `[More Information Needed]` where it's relevant ?